Cargo.toml

```diff
@@ -23,14 +23,14 @@ std = []
 async = []
 
 [target.'cfg(loom)'.dependencies]
-loom = "0.3.2"
+loom = "0.5"
 
 [dev-dependencies]
 criterion = "0.3"
 
 [target.'cfg(not(loom))'.dev-dependencies]
-tokio = { version = "0.2.20", features = ["rt-threaded", "macros", "time"] }
-async-std = { version = "1.5.0", features = ["attributes"] }
+tokio = { version = "1", features = ["rt", "macros", "time"] }
+async-std = { version = "1", features = ["attributes"] }
```

Comment:
Why is `async-std` changed downwards from 1.5 to 1?

Reply:
Happy to reintroduce the minor version though this will default to the latest version with `cargo build`.

Reply:
It's fine maybe.. Thinking about adding a CI job with `cargo update -Z minimal-versions` to make sure stuff like this does not break. But it's fine for now I guess.